torchvision/prototype/models/depth/stereo/raft_stereo.py

```diff
@@ -0,0 +1,710 @@
+from typing import List, Optional, Callable, Tuple
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torchvision.models.optical_flow.raft as raft
+from torch import Tensor
+from torchvision.models._api import WeightsEnum
+from torchvision.models.optical_flow._utils import make_coords_grid, grid_sample, upsample_flow
+from torchvision.models.optical_flow.raft import ResidualBlock, MotionEncoder, FlowHead
+from torchvision.ops import Conv2dNormActivation
+from torchvision.utils import _log_api_usage_once
+
+
+__all__ = (
+    "RaftStereo",
+    "raft_stereo",
+    "raft_stereo_fast",
+    "Raft_Stereo_Weights",
+    "Raft_Stereo_Fast_Weights",
+)
+
+
+def _corr_channel_normalize(corr: Tensor, num_channels: int):
+    return corr / torch.sqrt(torch.tensor(num_channels))
+
+
+def _zeroes_second_channel(delta_depth: Tensor):
+    delta_depth[:, 1] = 0.0
+
+
+torch.fx.wrap("_corr_channel_normalize")
+torch.fx.wrap("_zeroes_second_channel")
+torch.fx.wrap("make_coords_grid")
+torch.fx.wrap("grid_sample")
+
```

Comment:
We wrap some global functions (including one that is imported from raft) with `torch.fx.wrap` to make sure the model is fx compatible.